app/presenters/publishing_api_presenters/coming_soon.rb

```diff
@@ -13,15 +15,14 @@
 # Note this format becomes redundant once the caching infrasture is able to
 # honour caching headers on upstream 404 responses.
```

Comment:
Yes, after checking with Publishing Platform and Jamie, our infrastructure still doesn't honour caching headers on upstream 404 responses.